easybuild/easyconfigs/s/SU2-7.0.2-foss-2020a.eb

```diff
@@ -0,0 +1,57 @@
+# easyconfig file for SU2-7.0.6
+# By Alireza Ghavaminia a.ghavami@alumni.iut.ac.ir (Isfahan University of Technology)
+
+easyblock = 'ConfigureMake'
+
+name = 'SU2'
+version = '7.0.6'
```

Comment:
This file should have a `versionsuffix = '-Python-%(pyver)s'`
(also should be included in the filename)

Reply:
this as well -> 7.0.6